tools/run-dev.py

```diff
@@ -87,7 +87,7 @@
 if options.test:
     base_port = 9981
     settings_module = "zproject.test_settings"
-    # Don't auto-reload when running casper tests
+    # Don't auto-reload when running puppeteer tests
```

Comment:
Just leaving a note that we still need to update the `CASPER_TESTS` variable name to `PUPPETEER_TESTS`. I think while we at this, we can change this behavior to rely on a flag such as `--puppeteer-tests` rather than an environment variable.

Reply:
I think the reason we're using an environment variable is that lets us get the flag down to `zproject/test_extra_settings.py`, which doesn't have another convenient way to pass an option in.